lib/monitor/run.js

```diff
@@ -321,28 +327,11 @@ process.on('exit', function () {
 if (!utils.isWindows) {
   // usual suspect: ctrl+c exit
   process.once('SIGINT', function () {
-    var exitTimer = null;
-    var exit = function () {
```

Comment:
It's not obvious from the diff, but the removed code was functionally identical to the code in the `'quit'` handler. I wanted to remove the duplicated code for simplicity. See https://github.com/STRML/nodemon/blob/fixQuit/lib/monitor/run.js#L276